blog-detail.html

```diff
@@ -0,0 +1,350 @@
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Bakbak+One&family=Poppins&display=swap" rel="stylesheet">
+    </head>
+<body>
+    <header class="header">
+        <a href="#" class="logo-header-link">
+            <img src="./img/" alt="logo" class="logo">
+            BINABOX
+        </a>
+        <nav class="header-nav">
+            <ul class="header-nav-list">
+                <li class="header-nav-item">
+                    <a href="./index.html" class="header-nav-item-link">
+                        home
+                        <img src="./img/" alt="v" class="header-nav-item-img">
+                    </a>
+                </li>
+                <li class="header-nav-item">
+                    <a href="./about-us.html" class="header-nav-item-link">
+                        about
+                        <img src="./img/" alt="v" class="header-nav-item-img">
+                    </a>
+                </li>
+                <li class="header-nav-item">
+                    <a href="#" class="header-nav-item-link">
+                        roadmap
+                        <img src="./img/" alt="v" class="header-nav-item-img">
+                    </a>
+                </li>
+                <li class="header-nav-item">
+                    <a href="./catalog.html" class="header-nav-item-link">
+                        collection
+                        <img src="./img/" alt="v" class="header-nav-item-img">
+                    </a>
+                </li>
+                <li class="header-nav-item">
+                    <a href="./faqs.html" class="header-nav-item-link">
+                        faqs
+                        <img src="./img/" alt="v" class="header-nav-item-img">
+                    </a>
+                </li>
+                <li class="header-nav-item">
+                    <a href="#" class="header-nav-item-link">
+                        pages
+                        <img src="./img/" alt="v" class="header-nav-item-img">
+                    </a>
+                </li>
+            </ul>
+        </nav>
+        <div>
+            <button>
+                <img src="./img/" alt="mode_icon">
+            </button>
+            <a href="#" class="header-discord-link">
+                <img src="./img/" alt="discord_icon">
+                discord
+            </a>
+            <button type="button" class="header-connect-btn">
+                <img src="./img/" alt="connect_icon" class="header-connect-btn-img">>
+                connect
+            </button>
+        </div>
+    </header>
+    <main>
+        <section class="blog-detail-hero">
+            <div class="container">
+                <h1 class="blog-detail-title">BLOG DETAIL</h1>
+                <nav class="blog-detail-navigation">
```

Comment:
Could be improved: 
Homogeneous repeating elements (navigation; tags perceived as a list) should be implemented as a list - http://htmlbook.ru/html/ul